app/controllers/api/account.php

```diff
@@ -41,8 +41,8 @@
 use Utopia\Validator\Text;
 use Utopia\Validator\WhiteList;
 
-$oauthDefaultSuccess = '/v1/auth/oauth2/success';
-$oauthDefaultFailure = '/v1/auth/oauth2/failure';
+$oauthDefaultSuccess = '/auth/oauth2/success';
```

Comment:
This is actually `/auth/oauth/success` in the [Console](https://github.com/appwrite/console/tree/2.0.1/src/routes/auth/oauth/success). @ArmanNik, @TorstenDittmann, should we change it in the console code or here?

Reply:
Here's the PR if we want to go down the route of updating the path in the Console: https://github.com/appwrite/console/pull/170